sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -31,66 +32,69 @@
         ],
         "responses": {
           "200": {
-            "description": "Returns the Configuration object",
+            "description": "Returns the Configuration.",
             "schema": {
               "$ref": "#/definitions/Configuration"
             }
           }
         }
       },
-      "put": {
-        "summary": "Create or update the configuration for devices or modules of an IoT hub. An ETag must not be specified for the create operation. An ETag must be specified for the update operation. Note that configuration Id and Content cannot be updated by the user.",
-        "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
-        "operationId": "Configuration_CreateOrUpdate",
-        "consumes": [
-          "application/json"
-        ],
-        "produces": [
-          "application/json"
-        ],
-        "parameters": [
-          {
-            "name": "id",
-            "in": "path",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "configuration",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/Configuration"
-            }
-          },
-          {
-            "name": "If-Match",
-            "in": "header",
-            "required": false,
-            "type": "string"
-          },
-          {
-            "$ref": "#/parameters/api-version"
-          }
-        ],
-        "responses": {
-          "200": {
-            "description": "Returns the updated Configuration object",
-            "schema": {
-              "$ref": "#/definitions/Configuration"
-            }
-          },
-          "201": {
-            "description": "Returns the created Configuration object",
-            "schema": {
-              "$ref": "#/definitions/Configuration"
+        "put": {
+            "summary": "Create or update the configuration for Iot Hub device or module.",
```

Comment:
"... for your Iot Hub's devices and modules"?

Reply:
As someone who isn't well versed in these APIs, being a bit more specific as to who all these configurations apply to seems helpful

Reply:
Here is what I understood - You can update a Device or a module using this PUT. Is that incorrect and does the sentence not convey that?